### ISSUE TITLE
Enable Passive Transport for non-floating molecules

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite.Core#883]
Allow transport of non-floating molecules (enzymes, transporters, complexes).

My proposal is to allow the "non-floating" molecules to be transported by Passive Transports. By default, all non-floating molecules are excluded from every PT. Maybe the checkbox "Calculated for following molecules: All" whould be changed to "All floating", with an new (third) list where non-floating molecules can be included.

**Comment by Yuri05** [comment]
Is it still a requirement?
For the most floating molecules, passive transport will crash due to missing molecule parameters (partition coeffs etc.)

**Comment by StephanSchaller** [comment]
Well, yes. From OSMOSES :-)
The user is expected to know what parameters a molecule should have to be using a passive transport...

**Comment by PavelBal** [comment]
Yes, still a requirement. By default they are excluded, and the user has to explicitely add them.

**Comment by Yuri05** [comment]
why not just changing the molecule type to "floating" then?

**Comment by StephanSchaller** [comment]
> why not just changing the molecule type to "floating" then?

Major use-case internalizing drug-target complexes/Protein+binding partner (receptor+ligand). This might be required if we want to balance then the intracellular concentration as well / continue working with it intracellularly (and if internalization is not just a sink)

**Comment by Yuri05** [comment]
ok, so if you change the type of _drug-target complexes/Protein+binding partner_ to floating - they will be automatically transported by all passive transports.
If you need to exclude them from some passive transports - just add them to the transport exclude list.

Or do you see any problem with this solution?

**Comment by PavelBal** [comment]
> Or do you see any problem with this solution?

For most of the use cases, this would require excluding such a molecule from ALL passive transports, and adding a new passive transport for this molecule/complex.

Next problem - non-floaters and floaters require different parameters... Simply changing "Non-floating" to "floating" directly results in the errors:

![image](https://user-images.githubusercontent.com/25903040/207012944-9fb8f879-a73a-4bbf-bad5-e74ba86efd01.png)

(btw. what is the "no image data"?)

**Comment by Yuri05** [comment]
>Simply changing "Non-floating" to "floating" directly results in the errors:

nothing happens for me if I do this.
But I see your point in general.

**Comment by Yuri05** [comment]
For **active** transports would be easy change - just 1 line of code (did not think about possible side effects though).
@PavelBal What are actually your use cases for which PT of stationary molecules are required??

**Comment by PavelBal** [comment]
The same use case that Stephan mentioned - internalization of a receptor or receptor-ligand complext. For active transports, we would have to define an additional transporter protein for this, right?

**Comment by Yuri05** [comment]
>For active transports, we would have to define an additional transporter protein for this, right? 

Yes, you would need to create a dummy transporter protein which is present at the source and/or at the target container of the transported molecule.
